ansible-content-lab/aws_terraform_deployment
# Terraform template to create EC2 instance

This pr adds terraform template for creating EC2 instance.

Related Jira - https://issues.redhat.com/browse/AAP-18433

Steps to test - 
1. Pull down the pr
3. Deploy an EC2 instance using following steps mentioned here https://github.com/ansible-content-lab/aws_terraform_deployment#deploying-infrastructure

Expected Output - an EC2 instance should be created as per the provided configuration.
